proto/sdk/alpha/alpha.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC All Rights Reserved.
+// Copyright 2022 Google LLC All Rights Reserved.
```

Comment:
Non blocking: But there is no requirement to update copyright notices on edited files.

Reply:
OK. I'll revert this when other changes.

Reply:
Reverted the copyright notices